File: csharp/isogram/Isogram.cs
using System;
using System.Linq;

public static class Isogram
{
    public static bool IsIsogram(string word) =>
        word.ToLowerInvariant().ToCharArray()
            .Where(c => Enumerable.Range('a', 26).Contains(c))
            .GroupBy(c => c)
            .Where(g => g.Count() > 1)
            .Count() == 0;
}

File: fsharp/isogram/Isogram.fs
module Isogram

let isIsogram (str: string) = 
    str.ToLowerInvariant()
    |> Seq.filter "abcdefghijklmnopqrstuvwxyz".Contains
    |> Seq.countBy (fun c -> c) 
    |> Seq.forall (fun (c, x) -> x = 1)